e2e/poms/dashboard.ts

```diff
@@ -0,0 +1,44 @@
+import { Page, expect } from "@playwright/test";
+import { DASHBOARD_SELECTORS } from "../constants/selectors";
+import { DASHBOARD_LINKS } from "../constants/links/dashboard";
+
+export default class Dashboard {
+    constructor(private page: Page) {
+        this.page = page;
+    }
```

Comment:
```suggestion
    constructor(private page: Page) {
    }
```

There is a shorthand property in TypeScript where you can avoid initialization of class variable with paramater values in constructor by simply using the visibility in the constructor. The above code is equivalent to:

```typescript
class Dashboard {
    page: Page

    constructor(page: Page) {
      this.page = page;
    }
}
```